src/test/java/com/bizo/awsstubs/services/dynamodb2/AmazonDynamoDBStubTest.java

```diff
@@ -380,11 +379,26 @@ private PutItemResult putItem(String attributeName, String attributeValue) throw
     return result;
   }
 
-  @Test(expected=InternalServerErrorException.class)
+  @Test
   public void test_query() throws Exception {
```

Comment:
Thanks for including a test!

Maybe for good measure just add another test that has two items, one that matches and one that doesn't, and see only the matched item come back?